_includes/jobs.html

```diff
@@ -10,7 +10,12 @@ <h1>Open positions</h1>
                 <div class="flip-card-front">
                   <p>
                     <h3>{{job.['Job Title']}}</h3>
-                    <h3 class="location">{{job.Location}}</h3>
+                    {% assign location = job.Location%}
+                    {% if location == "Global" %}
+                      <h3 class="location">{{location}}</h3>
+                    {% elsif location != "Global"%}
+                      <h3 class="location">{{location | split: ", " | join:"<br />"}}</h3>
+                    {% endif %}
```

Comment:
Instead of checking for the `Global` string, can't we always apply the splitting behaviour? If it's `Global` the splitting will produce the same result 👍 . I think it makes it clearer and it's easier to reason with.

Reply:
Ok